configuration/analyzer_config.json

```diff
@@ -1826,6 +1826,23 @@
       }
     }
   },
+  "PassiveDNS" : {
+    "type" : "observable",
+    "python_module" : "passivedns.passiveDNS",
+    "description" : "Look up the PDNS information of a domain using the PassiveDNS API.",
+    "disabled" : false,
+    "external_service" : true,
+    "leaks_info" : false,
+    "observable_supported": [
+      "domain"
+    ],
+    "config" : {
+      "soft_time_limit" : 30,
+      "queue" : "default"
+    },
+    "secrets" : {},
+    "params" : {}
```

Comment:
add support for `limit` param.
![Uploading image.png…]()

Reply:
Also,

their API provides two endpoints which return result in different formats.
- https://api.mnemonic.no/pdns/v3/cof/google.com
- https://api.mnemonic.no/pdns/v3/google.com

for this, we should provide another param `cof_format` (default: `true`) which switches the URL.